.travis.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         - xcodebuild $XCODE_ACTION -workspace "$XCODE_WORKSPACE" -scheme "$XCODE_SCHEME" -sdk "$XCODE_SDK" -destination "$XCODE_DESTINATION" -derivedDataPath "$XCODE_DERIVED_DATA_PATH" | xcpretty
       after_success:
         - bash <(curl -s https://codecov.io/bash) -D $XCODE_DERIVED_DATA_PATH -J "^$XCODE_SCHEME$"
+    - stage: SwiftPM Build
+      env:
+        - PLATFORM_TARGET=x86_64-apple-ios13.0-simulator
```

Comment:
I'm assuming that we're using 13 due to `osx_image` value. Couldn't we bump it up or would it be better to do on another pr?

Reply:
You're assuming correctly 😉 And yes, I think that we should do that bump (Swift version, Xcode) in a separate PR